src/ESLint.Formatter/sarif.js

```diff
@@ -175,7 +191,10 @@ module.exports = function (results, data) {
                                 };
                                 if (meta.docs.description) {
```

Comment:
>meta.docs.description [](http://example.com/codeflow?start=36&length=21)

what are the other fields we have in meta.docs? what is a meta?

Reply:
meta represents rule meta data, below is an example:

"no-undef": {
    "type": "problem",
    "docs": {
        "description": "disallow the use of undeclared variables unless mentioned in `/*global */` comments",
        "recommended": true,
        "url": "[[https://eslint.org/docs/rules/no-undef](https://eslint.org/docs/rules/no-undef)](https://eslint.org/docs/rules/no-undef)"
    },
    "schema": [
        {
            "type": "object",
            "properties": {
                "typeof": {
                    "type": "boolean",
                    "default": false
                }
            },
            "additionalProperties": false
        }
    ],
    "messages": {
        "undef": "'{{name}}' is not defined."
    }
},